wire-ios-request-strategy/Sources/Request Strategies/User/SelfUserRequestStrategy.swift

```diff
@@ -23,20 +23,16 @@ public final class SelfUserRequestStrategy: AbstractRequestStrategy {
     // MARK: - Properties
 
     private let transcoder = Transcoder()
-    private let upstreamSync: ZMUpstreamModifiedObjectSync
+    private let actionSync: EntityActionSync
 
     // MARK: - Life cycle
 
     public override init(
         withManagedObjectContext managedObjectContext: NSManagedObjectContext,
         applicationStatus: ApplicationStatus
     ) {
-        upstreamSync = ZMUpstreamModifiedObjectSync(
-            transcoder: transcoder,
```

Comment:
Don't forget to also delete the now dead upstream transcoder methods in this file.